### PR TITLE
Christina development

### PR DESCRIPTION
**Pull request by christinachance**

### CODE REVIEW

File: portfolio/src/main/webapp/blog_script.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            commentHistory.appendChild(createListElement(line));
+        });
+    });
+}
+
+function createListElement(text) {
+  const liElement = document.createElement('li');
+  liElement.innerText = text;
+  return liElement;
+}
```
Comment (serzhk) [inline portfolio/src/main/webapp/blog_script.js:30]: empty line at the end please.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -14,19 +14,36 @@
...
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException{
+    String comment = request.getParameter("comment");
+    commentSection.add(comment);
+    response.sendRedirect("/blog.html");
+  }
+  private String getComment(HttpServletRequest request) {
+    String comment = request.getParameter("comment");
+    return comment;
```
Comment (serzhk) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:47]: You could directly: return request.getParameter("comment");

Comment (christinachance) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:47]: done.

File: portfolio/src/main/webapp/blog_script.js (hunk 1 above)
Comment (christinachance) [reply portfolio/src/main/webapp/blog_script.js:30]: done.

**Pull request merged**